packages/blade/src/utils/testing/renderWithSSR.web.tsx

```diff
@@ -20,17 +25,43 @@ const renderWithSSR = (ui: ReactElement): RenderResult => {
   );
 
   const { JSDOM } = jsdom;
-  const dom = new JSDOM(html);
-  // @ts-expect-error ignoring this error since we are trying to make tests work between node and jsdom environments
-  global.window = dom.window;
-  global.document = dom.window.document;
+  const dom = new JSDOM(html, { pretendToBeVisual: true, url: 'http://localhost/' });
```

Comment:
was this the only issue?

Reply:
Check the PR description the first two points that I've mentioned were majorly the issue.